sw/device/tests/sim_dv/lc_ctrl_program_error.c

```diff
@@ -59,8 +60,7 @@ bool test_main(void) {
       &alert_handler));
 
   // Check if there was a HW reset caused by the escalation.
-  dif_rstmgr_reset_info_bitfield_t rst_info;
-  CHECK_DIF_OK(dif_rstmgr_reset_info_get(&rstmgr, &rst_info));
+  dif_rstmgr_reset_info_bitfield_t rst_info = rstmgr_testutils_reason_get();
   CHECK_DIF_OK(dif_rstmgr_reset_info_clear(&rstmgr));
```

Comment:
This is already cleared. You may want to clear the reset reason here.

Reply:
If it is already cleared, do you mean there is no need to clear it here?

Reply:
Done.